templates/home.html

```diff
@@ -54,4 +54,4 @@ <h2 class="wp-block-heading">Posts</h2>
   <!-- /wp:columns --></div>
   <!-- /wp:group -->
 
-  <!-- wp:template-part {"slug":"footer"} /-->
```

Comment:
I am not sure what this is. I am guessing VS Code removed an empty line at the end of the file?